docs/src/pages/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+
+import { LandingHero } from "../components/landing/landing-hero";
+import { LandingBanner } from "../components/landing/landing-banner";
+import { LandingFeaturedProjects } from "../components/landing/landing-featured-projects";
+import { LandingFeatures } from "../components/landing/landing-features";
+import SwipeDemo from "../components/examples/SwipeDemo";
+import {
+  feature1,
+  feature2,
+  feature3,
+} from "../components/landing/landing-images";
+
+export default function Home(): JSX.Element {
+  const { siteConfig } = useDocusaurusContext();
+  return (
+    <Layout title={siteConfig.title} description={siteConfig.tagline}>
+      <div className="dark:bg-gray-500 bg-gray-200 dark:text-white text-theme-2">
+        <LandingHero
+          heading={siteConfig.title}
+          body={siteConfig.tagline}
+          copyText="npm add react-swipeable"
+          navItems={[
+            { link: "/docs", title: "Documentation" },
+            { link: "/docs/examples/simple-carousel", title: "Demo" },
+            {
+              link: "https://github.com/FormidableLabs/react-swipeable",
+              title: "Github",
+            },
+          ]}
+        ></LandingHero>
+      </div>
+      <LandingFeatures
+        heading="Features"
+        list={[
+          {
+            imgSrc: feature1,
+            alt: "robust",
+            title: "Robust",
+            html: {
+              __html:
```

Comment:
🤔 what can go here aside from the double mention of `useSwipeable` as 2 of 3 features? In your update did anything else come to mind that could be used here?